packages/xo-server/src/xapi/mixins/storage.js

```diff
@@ -84,4 +84,28 @@ export default {
     })
     return unhealthyVdis
   },
+
+  async createSr({
+    host,
```

Comment:
`Xapi` methods should receive XAPI objects (or UUIDs or refs) but not XO objects.